Update mirroring.md - incident IDs returned by get-modified-remote-data must be of type str (#534)
demisto/content-docs: docs/integrations/mirroring.md

* **GetModifiedRemoteDataArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
Arguments explanation:
  - *last_update* - Date string represents the last time we retrieved modified incidents for this integration.
* **GetModifiedRemoteDataResponse** - this is the object that maintains the format in which you should order the results from this function. You should use `return_results` on this object to make it work.
Arguments explanation. **You must provide one of the following:**

>   - *modified_incident_ids* - a list of the IDs of the incidents that were modified since the last check. IDs must be of type string. Later `get-remote-data` command will run on only modified incidents.

Reviewer: ```suggestion
  - *modified_incident_ids* - a list of strings representing incident IDs that were modified since the last check. Later `get-remote-data` command will run on only modified incidents.
```
